### PR TITLE
Aida 401

### PR DESCRIPTION
**Pull request by gitnordt**

### CODE REVIEW

File: src/app/components/legend/legend.component.ts [hunk 1]
```diff
@@ -98,8 +98,8 @@ export class LegendComponent implements OnInit {
     }
 
     ngOnInit() {
-        this.loadAllColorSets();
-    }
+            this.loadAllColorSets();
```
Comment (bdpham) [inline src/app/components/legend/legend.component.ts]: Looks like indentation is off?

**Comment by bdpham** [comment]
Looks like this branch has commits from previously closed PR's, which makes it a bit difficult to differentiate between the changes you've made vs the changes that have already been pushed in.

**Comment by ThomasSchellenbergNextCentury** [comment]
I THINK @snorpet wanted to merge her feature branch into verdi-master but was accidentally trying to merge it into master.  That's why @bdpham was confused about commits from previously closed PRs.  I've updated the base branch of this PR to correct this issue.

File: src/app/components/network-graph/network-graph.component.ts [hunk 2]
```diff
@@ -678,13 +674,6 @@ export class NetworkGraphComponent extends BaseNeonComponent implements OnInit,
         //console.log('Item clicked', data);
     }
 
-    /*
-    setColorScheme(name) {
-        this.selectedColorScheme = name;
-        this.colorScheme = this.colorSets.find(s => s.name === name);
-    }
-    */
```
Comment (ThomasSchellenbergNextCentury) [inline src/app/components/network-graph/network-graph.component.ts:677]: Why is this function just commented out?  If we think we're going to need it in the future, could we add an appropriate JIRA ticket to the comment?  If we don't think we're going to need it, can we just delete it?

File: src/app/components/aggregation/aggregation.component.spec.ts [hunk 3]
```diff
@@ -14,4909 +14,1056 @@
...
 
-describe('Component: Aggregation', () => {
-    let component: AggregationComponent;
-    let fixture: ComponentFixture<AggregationComponent>;
-    let getService = (type: any) => fixture.debugElement.injector.get(type);
-
-    let COLOR_1 = new Color(173, 216, 230);
-    let COLOR_2 = new Color(228, 26, 28);
-    let COLOR_3 = new Color(55, 126, 184);
+describe('Component: BarChart', () => {
```
Comment (ThomasSchellenbergNextCentury) [inline src/app/components/aggregation/aggregation.component.spec.ts]: So it looks like somehow you overwrote the Aggregation Component tests in aggregation.component.spec.ts with new Bar Chart Component tests?  Should they have gone in bar-chart.component.spec.ts?

**Comment by gitnordt** [comment]
> Looks like this branch has commits from previously closed PR's, which makes it a bit difficult to differentiate between the changes you've made vs the changes that have already been pushed in.

@bdpham @ThomasSchellenbergNextCentury  That's weird. I can only see the 5 commits I did. I cannot see commits from a previously closed PR.

File: src/app/components/legend/legend.component.ts (hunk 1 above)
Comment (gitnordt) [reply src/app/components/legend/legend.component.ts]: Fixed

**Comment by gitnordt** [comment]
> > Looks like this branch has commits from previously closed PR's, which makes it a bit difficult to differentiate between the changes you've made vs the changes that have already been pushed in.
> 
> @bdpham @ThomasSchellenbergNextCentury That's weird. I can only see the 5 commits I did. I cannot see commits from a previously closed PR.

@bdpham @ThomasSchellenbergNextCentury Oh, Thomas fixed it! Yaaay!

**Comment by bdpham** [comment]
@snorpet Thomas fixed it by rebasIng the branch.

File: src/app/components/network-graph/network-graph.component.ts (hunk 2 above)
Comment (gitnordt) [reply src/app/components/network-graph/network-graph.component.ts:677]: This has already been deleted.

File: src/app/components/aggregation/aggregation.component.spec.ts (hunk 3 above)
Comment (gitnordt) [reply src/app/components/aggregation/aggregation.component.spec.ts]: These are not my changes. These are the changes that I merged in from your PR THOR-851 to address the colorMap issue.

Comment (ThomasSchellenbergNextCentury) [reply src/app/components/aggregation/aggregation.component.spec.ts]: Hmm, I'm not sure what happened but it seems to have fixed itself.  Yay!

**Review by bdpham — APPROVED** [review]

**Review by ThomasSchellenbergNextCentury — APPROVED** [review]

**Pull request merged**